src/wallet.py

```diff
@@ -0,0 +1,86 @@
+import json
+from json.decoder import JSONDecodeError
+
+import structlog
+from nacl.encoding import HexEncoder
+from nacl.signing import SigningKey
+import base64
+from cryptography.fernet import Fernet
+
+
+logger = structlog.get_logger(__name__)
+
+class Wallet:
+    def __init__(self): 
+        self.privatekey = ""
```

Comment:
As you suggested in another comments, perhaps we can have only the file containing  the key? So that its a bit more secure. But up to you.